### PR TITLE
Add Cookies to `users.md`

### PR DESCRIPTION
**Pull request by sgammon**
We are big adopters of Bazel at Cookies :)

### CODE REVIEW

File: users.md [hunk 1]
```diff
@@ -60,6 +60,17 @@ CarGurus is on a mission to build the world's most trusted and transparent autom
 
 Compass is a tech-driven real estate platform. With an elite team of real estate, technology and business professionals, we aim to be the best and most trusted source for home seekers.
 
+### [Cookies](https://cookies.co)
+
+<img src="https://img.cookies.co/static/logo/cookies-blue.png" width="150" align="right">
+
+COOKIES is one of the most well-respected and top-selling cannabis brands in California and throughout the world. The company and its product are recognized globally and offer a stable of over 85 cannabis varieties and product lines including indoor and sungrown flower, pre-rolls, gel caps, and vape carts. COOKIES' seed-to-sale business allows for complete quality control at every step—from cultivation and production to customers' end retail experience. Cookies has expanded to 33 stores in seven states and three countries, with 20+ more planned to open by the end of 2021.
```
Comment (aiuto) [inline users.md:67]: This feels a little too market-y.
And let's keep it time-insensitive, so no plans for the future, unless they are about Bazel.

**Comment by hicksjoseph** [comment]
This is, perhaps, too much company info. Perhaps something like this would be better, "COOKIES is one of the most well-respected and top-selling cannabis brands in California and throughout the world. Eng@Cookies and our Digital Platform are powered by a central Bazel monorepo, with components written in Java, Kotlin, Typescript and more. Services are powered by gRPC, and specified via the OpenCannabis stack of specifications, which are also built with Bazel."

File: users.md (hunk 1 above)
Comment (sgammon) [reply users.md:67]: will fix :) thank you for the review

**Comment by sgammon** [comment]
@hicksjoseph / @aiuto thank you! i've amended to your suggested copy. you're a better english programmer than i am 😁

**Review by hicksjoseph — APPROVED** [review]
LGTM

**Pull request merged**